events-api.go

```diff
@@ -0,0 +1,107 @@
+package kable
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/Kable-io/kable-go/internal/openapi"
+	"github.com/ehsaniara/gointerlock"
+	"io"
+	"log"
+	"time"
+)
+
+type (
+	Event openapi.Event
+)
+
+type EventsApi struct {
+	api     *openapi.APIClient
+	options *KableOptions
+	ctx     context.Context
+	queue   []Event
+}
+
+type RecordEventOut struct {
```

Comment:
Can this be removed as well? This is an internal response structure from the Kable events API, right? 

I don't think we even advertise a response body in the [docs](https://docs.kable.io/reference/createevents). If it's possible, let's not expose a response body at all here -- whatever Go's version of `void` is would be great

Reply:
Sure. We can remove this.